src/components/Input.scss

```diff
@@ -158,5 +159,39 @@ $form-input-bg-color: $cu-foreground--dark !default;
       @include ss-pika;
     }
   }
+}
 
+// * 5. Errors and feedback
+.Input-feedback {
+  display: none;
+}
+.is-required.is-invalid {
```

Comment:
I'd still like to see these classes attached to whatever class they are modifying.

Reply:
Should I update elsewhere in the file as well? Like here: 

.is-required {
  .Input-label--inline:after {
    display: none;
  }
  .Input-label--inline:before {
    content: '*';
    display: inline-block;
    padding-right: $su-xsmall;
    color: $cu-info--light;
  }
}

And here:

.is-required {
  .Input-label:after {
    content: '*';
    display: inline-block;
    padding-left: $su-xsmall;
    color: $cu-info--light;
  }
}

Reply:
Yeah, these should all be modifying a class. If you want to update it that would be great.